crates/figma_import/src/bin/fetch.rs

```diff
@@ -86,14 +90,18 @@ fn fetch_impl(args: Args) -> Result<(), ConvertError> {
         id: doc.get_document_id(),
     };
     println!("Fetched document");
+    println!("  DC Version: {}", SerializedDesignDocHeader::current().version);
     println!("  Doc ID: {}", doc.get_document_id());
     println!("  Version: {}", doc.get_version());
     println!("  Name: {}", doc.get_name());
     println!("  Last Modified: {}", doc.last_modified().clone());
     // We don't bother with serialization headers or image sessions with
     // this tool.
-    let output = std::fs::File::create(args.output)?;
-    bincode::serialize_into(output, &serializable_doc)?;
+    let mut output = std::fs::File::create(args.output)?;
+    let header = bincode::serialize(&SerializedDesignDocHeader::current())?;
```

Comment:
Is there common code that can be reused here?

Reply:
Not really -- it's just one line. If it gets more involved or used more I'll separate it out.